### PR TITLE
TLSUV rename: phase 2

### PR DESCRIPTION
**Pull request by ekoby**

### CODE REVIEW

File: include/tlsuv/tlsuv.h [hunk 1]
```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2019-2020 NetFoundry, Inc.
```
Comment (dovholuknf) [inline include/tlsuv/tlsuv.h:2]: copyright change

File: include/tlsuv/tlsuv.h [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+const char* tlsuv_version();
+
+typedef struct tlsuv_stream_s tlsuv_stream_t;
+
+typedef void(*tlsuv_log_func)(int level, const char *file, unsigned int line, const char *msg);
```
Comment (dovholuknf) [inline include/tlsuv/tlsuv.h:36]: doc?

File: include/tlsuv/http_src_t.h [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+// forward ref
+typedef struct tlsuv_http_s tlsuv_http_t;
+
+/**
+ * Source link types
+ */
+typedef struct tlsuv_src_s tlsuv_src_t;
+
+typedef void (*tlsuv_src_connect_cb)(tlsuv_src_t *sl, int status, void *connect_ctx);
```
Comment (dovholuknf) [inline include/tlsuv/http_src_t.h:40]: doc?

File: sample/CMakeLists.txt [hunk 4]
```diff
@@ -1,15 +1,11 @@
...
 target_link_libraries(sample PUBLIC tlsuv)
-target_include_directories(sample PUBLIC ${CMAKE_SOURCE_DIR}/include)
 
 add_executable(engine_test engine_test.c)
 target_link_libraries(engine_test PUBLIC tlsuv)
-target_include_directories(engine_test PUBLIC ${CMAKE_SOURCE_DIR}/include)
 
 add_executable(sample-cf sample-cf.c)
 target_link_libraries(sample-cf PUBLIC tlsuv)
-target_include_directories(sample-cf PUBLIC ${CMAKE_SOURCE_DIR}/include)
```
Comment (dovholuknf) [inline sample/CMakeLists.txt:12]: all these removes are on purpose i expect, but calling out to make sure

**Review by dovholuknf — APPROVED** [review]
i think you should add doc to the .h files and there's a copyright issue to fix and check the sample/CMakeLists.txt for the removals being on purpose

File: sample/CMakeLists.txt (hunk 4 above)
Comment (ekoby) [reply sample/CMakeLists.txt:12]: yes

**Pull request merged**